### ISSUE TITLE
Default installation directory

### ISSUE THREAD

**Issue by ignace72** [eoyilmaz/displaycal-py3#127]
Hello, @eoyilmaz,
DisplayCAL 3.9.5 is installed in the /usr/ directory when it is rather reserved for package installations.
It would be better to use the /usr/local/ directory which is made for that.
Thank you.

**Comment by eoyilmaz** [comment]
Hi @ignace72 thank you for reporting. But, how did you install DisplayCAL?

**Comment by ignace72** [comment]
For my installation, I made it in my personal directory in .local/.
At the installation (pip install diplaycal), a window opens asking for the password. Instead of entering my password, I clicked on the "Cancel" button.

**Comment by ignace72** [comment]
Basically, there are three directories available for installations:
The /usr/ directory for system-dependent software (with the package manager).
The /usr/local/ directory for locally compiled software or software installed outside the package manager but conforming to the standard UNIX directory organization.
And the /opt/ directory with third party software that does not respect the organization of UNIX directories, such as proprietary software or, in our case, DisplayCAL 3.9.1, 3.9.2, 3.9.3 and 3.9.4.
The DisplayCAL packages in Python 3 for ArchLinux were installed in /usr/ because the files were stored in the standard UNIX directories.

**Comment by eoyilmaz** [comment]
I checked your webpage (without translating it :) ) as I understand, you are claiming that even though you are using ``pip install displaycal`` in a virtualenv our installer puts the files in to ``/usr/`` directory, is that the case?

**Comment by ignace72** [comment]
Hello, @eoyilmaz.
Thank you for the effort of reading my page without translating it.
In the link I gave you, the page is dated May 16, so I don't talk about the installation with "pip install displaycal", because this installation didn't exist (if I understood correctly, it's a new feature of the current version).
If you click on an internal link, you will be taken back to the current version of the page (from May 30th).

I tried the installation of 3.9.5 in a virtual environment and the installation is done in the virtual environment directory.
I did the installation as follows:
```
pip3 install virtualenv
pip3 install wheel
python3 -m virtualenv ./displaycal_venv && source ./displaycal_venv/bin/activate
pip install displaycal
```
And launch the program with a :
`/home/ignace/Downloads/displaycal/displaycal/displaycal_venv/bin/displaycal`

Personally, with the new way of installing DisplayCAL, I don't see the point of using a virtual environment, because it works fine without it.

If you have any criticism on my page (the current version), don't hesitate. I created this page with the greatest respect for your work so it would bother me if you didn't like something.

**Comment by eoyilmaz** [comment]
The point of having a virtual environment is to not to make the system Python dirty. I think, in near future we'll have proper packages for each mainstream Linux OSes, MacOS and Windows.

What I was trying to ask was, if you were getting some files installed in to ``/usr`` even though you were using ``pip install displaycal``. To be honest, this should not be possible, because we are using Python wheel. But, there are some functions run by the installer to move some files to some certain locations in the system. I thought, maybe one of the functions are not working properly and installing moving files to the system folders instead of the virtual environment folders.

And, no worries, you've done a great job on explaining things :+1:

**Comment by ignace72** [comment]
Good evening, eoyilmaz.
With the method I described above, the files are all installed in the `displaycal_venv` directory. pip can't install anything in `/usr/` anyway because it hasn't been given write permissions to that directory.
The virtual environment was taken into account since pip installed the software in the `displaycal_venv` directory and not in `.local/`. If I understood correctly how the virtual environment for Python works, pip has no reason to install the software somewhere else.
The files are either installed in `/usr/`, `.local/` or `displaycal_venv` according to the same tree structure: `bin/`, `lib/` and `share/`.
In the three locations, there are the same files, I did not notice that any were missing during an installation without root rights.
I imagine that if something was missing, pip would have put an error message in the terminal.
I hope to have answered your question.
Thank you for your review on my page. I'm glad you like my work. I'm not a developer, just an avid user of GNU/Linux since 1995, so I help as best I can by promoting the software and how to install it on many French speaking forums to amateur photographers and users GNU/Linux who need display calibration software because it is an imperative step for a quality workflow and before DisplayCAL in Python 3 appeared, we were deprived of this step. Well, I know, we can do the calibration with ArgyllCMS on the command line but the majority of photographers and artists in general are not computer scientists and are resistant to the command line.
Installing DisplayCAL for Python 3 from the command line is easier than doing the calibration from the command line. Even more with your version 1.9.5 since there is only one command line, so the installation has become accessible even to the most recalcitrant.
Sorry if the English of my message is of lower quality but I used Google translation because I reached the free limits of DeepL and I still wanted to finish my message.
Thank you.
